admin/installer/cloudformation.py

```diff
@@ -70,19 +70,29 @@ def _sibling_lines(filename):
 # Keys corresponding to CloudFormation user Inputs.
 access_key_id_param = template.add_parameter(Parameter(
     "AmazonAccessKeyID",
-    Description="Your Amazon AWS access key ID",
+    Description="Your Amazon AWS access key ID (mandatory)",
     Type="String",
+    NoEcho=True,
+    AllowedPattern="[\w]+",
```

Comment:
:question: Maybe add a ConstraintDescription here too saying "can contain only alphanumeric characters and  underscores" (at least that's how the Python RE docs describe `\w`) 
 * https://docs.python.org/2/library/re.html#regular-expression-syntax

We could also link to https://docs.aws.amazon.com/IAM/latest/APIReference/API_AccessKey.html in the constraint description.